qemu/tests/cfg/rng_read_longtime.cfg

```diff
@@ -0,0 +1,27 @@
+- rng_read_longtime:
+    type = rng_bat
+    kill_vm = yes
+    virt_test_type = qemu
+    read_rng_timeout = 3600
+    no no_virtio_rng
+    test_duration = 36000
+    Windows:
+        # Please update path of rng_dll_register_cmd to right path which included you driver
+        #rng_dll_register_cmd = if not exist "C:\Windows\system32\viorngum.dll" copy PATH:\INCLUDEDRIVER\viorngum.dll C:\Windows\system32\ /y &&"
+        session_cmd_timeout = 240
+        read_rng_cmd  = for /l %i in (1, 1, 30) do "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
```

Comment:
it looks here should be 
for /L %%i in (1,1,30) do (your command)
Please have a try.